prow/plugins/label/label.go

```diff
@@ -128,13 +128,10 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 }
 
 func (ae assignEvent) getRepeats(sigMatches [][]string, existingLabels map[string]string) (toRepeat []string) {
-
 	toRepeat = []string{}
 	for _, sigMatch := range sigMatches {
 		sigLabel := strings.ToLower("sig" + "/" + strings.TrimSpace(sigMatch[1]))
```

Comment:
a question unrelated to this PR: do we need to check the length of `sigMatch` before using it?